dynamic-neural-field-composer/scripts/bench.ps1

```
@@ -0,0 +1,131 @@
+
+$HighPerfGuid = '8c5e7fda-e8bf-4a96-9a85-a6e23a8c635c'
+$originalGuid = $null
+try {
+    $schemeLine = (powercfg /getactivescheme) -join "`n"
+    if ($schemeLine -match '([0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12})') {
+        $originalGuid = $Matches[1]
+    }
+}
+catch {
+    # powercfg itself is missing or unusable -- proceed unpinned on clocks and say so.
+}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)